extensions/cache/src/main/java/org/radargun/stages/cache/background/BackgroundStressorLogic.java

```diff
@@ -19,7 +19,7 @@
  * @author Radim Vansa &lt;rvansa@redhat.com&gt;
  * @author Michal Linhard &lt;mlinhard@redhat.com&gt;
  */
-class LegacyLogic extends AbstractLogic {
+class BackgroundStressorLogic extends AbstractLogic {
```

Comment:
I think you were too eager in renaming this - LegacyLogic in background ops tests is not related to legacy/experimental tests.